core/integration/module_typescript_test.go

```diff
@@ -1779,3 +1779,37 @@ class Test {
 		require.JSONEq(t, `{"test": {"foo": "bar"}}`, out)
 	})
 }
+
+func (TypescriptSuite) TestPrivatePropertyWithUnsupportedType(ctx context.Context, t *testctx.T) {
+	c := connect(ctx, t)
+
+	modGen := c.Container().From(golangImage).
+		WithMountedFile(testCLIBinPath, daggerCliFile(t, c)).
+		WithWorkdir("/work").
+		With(daggerExec("init", "--name=test", "--sdk=typescript")).
+		With(sdkSource("typescript", `import { object, func } from '@dagger.io/dagger'
+
+@object()
+export class Test {
+	@func()
+	foo: string
```

Comment:
We don't need this property in the test.

Reply:
True, I removed it